specification/compute/resource-manager/readme.md

```diff
@@ -151,6 +151,19 @@ directive:
       - TrackedResourceGetOperation
       
 ```
+### Tag: package-2018-04-2

```

Comment:
@hyonholee Is there a reason why it doesn't match API version?

- API version: 2018-04-01
- Tag: package-2018-04-2

Reply:
@sergey-shandar Oh, I just set the tag name to "package-2018-04-2" because "package-2018-04" already exists.  Could you suggest a better tag name?

Reply:
@hyonholee Oh, I see. It looks like `2018-04-01/disk.json` is shared. In this case, it's breaking changes, right? Do you change service behavior for `disk` api version `2018-04-01`? Since the `2018-04-01` disk is already in stable folder, I would recommend to create a new api version in this case.